conda/recipes/cucim/build.sh

```diff
@@ -11,13 +12,13 @@ export LD_LIBRARY_PATH="$BUILD_PREFIX/lib:$PREFIX/lib:$LD_LIBRARY_PATH"
 
 # It is assumed that this script is executed from the root of the repo directory by conda-build
 # (https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake)
-./run build_local cucim ${CUCIM_BUILD_TYPE}
+./run build_local cucim "${CUCIM_BUILD_TYPE}"
 
 cp -P python/install/lib/* python/cucim/src/cucim/clara/
 
-pushd python/cucim
+pushd python/cucim || exit 1
```

Comment:
Would `shellcheck` stop complaining about this if we did a `set -e` at the top of this script?

If so, I think that'd be preferable. And honestly either way, I think it'd be helpful for this to get `set -e -u -o pipefail` added to it, so failures in e.g. the `pip install` below actually result in failures in CI. Could you add that please?

Reply:
Yep, good call